packages/client/src/actions/group.e2e.ts

```diff
@@ -0,0 +1,60 @@
+import type { EvmAddress } from '@lens-chain/storage-client';
+import { group } from '@lens-protocol/metadata';
+import { assertOk, nonNullable, uri } from '@lens-protocol/types';
+import { beforeAll, describe, it } from 'vitest';
+import {
+  loginAsAccountOwner,
+  loginAsBuilder,
+  TEST_ACCOUNT,
+  wallet,
+} from '../test-utils';
+import { handleOperationWith } from '../viem';
+import { createGroup, fetchGroup, joinGroup } from './group';
+
+const metadata = group({
+  name: 'web3natives',
+});
+
+describe('Given a logged-in user', () => {
+  describe('When joining a Group with SimplePaymentRule in native token denomination', () => {
+    let groupAddress: EvmAddress = '0x9904A9E96dfB83409Cd0D89DF15588eDBb99B322';
+
+    beforeAll(async () => {
+      const result = await loginAsBuilder().andThen((sessionClient) =>
+        createGroup(sessionClient, {
```

Comment:
do we want to create a group each run of the tests? if so then we should remove the default address (test will fail anyway if the beforeAll throws error.